components/Controllers/Loans/Loans.tsx

```diff
@@ -23,7 +23,8 @@ export function Loans() {
   const { pricesData } = useControllerPricesData();
   const paprController = useController();
   const currentVaults = useMemo(
-    () => paprController.vaults?.filter((v) => v.debt > 0),
+    () =>
+      paprController.vaults?.filter((v) => v.debt > 0 && v.collateralCount > 0),
```

Comment:
could probably do with just the latter, unless we've made a serious mistake! but 🤷

Reply:
in this case the one with no collateral did show some debt